src/app/security/bug-bounty/data/leaderboardTableColumnDef.tsx

```diff
@@ -5,13 +5,13 @@ import { createColumnHelper } from '@tanstack/react-table'
 
 import { Icon } from '@/components/Icon'
 
-import { leaderboardData } from '../data/leaderboard'
+import { leaderboardData } from './leaderboard'
 
 type WhiteHat = (typeof leaderboardData)[number]
 
 const columnHelper = createColumnHelper<WhiteHat>()
 
-export const leaderboardTableData = [
+export const leaderboardTableColumnDef = [
```

Comment:
This code is closely tied to UI rendering, as it defines table columns and handles icon rendering for profile links, which are part of the component’s structure. Since it’s specific to how the table and icons are displayed, I believe it belongs in the `components` folder rather than the `data` folder to keep UI logic organized. I would also rename it to `leaderboardColumns`. I also believe we already made this move once..

Reply:
Well, this is hybrid; it contains both data and UI code.

But, if I had to choose, I’d say it looks more like data than components. It’s an array of config objects, not a function returning JSX. We already do something similar in a few places, like `grants/data/applicationProcessData.tsx`, for instance.

That’s just my two cents on the matter, your call :)

Reply:
Yes ~ Better like that! ❤️

Reply:
Ok, let's do that then :)